src/model/immutable/EditorState.js

```diff
@@ -377,16 +377,22 @@ class EditorState {
       );
     }
 
-    return EditorState.set(editorState, {
+    var editorStateChanges = {
       currentContent: newContent,
       directionMap,
       undoStack,
       redoStack: Stack(),
       lastChangeType: changeType,
       selection: contentState.getSelectionAfter(),
-      forceSelection,
-      inlineStyleOverride: null,
-    });
+      forceSelection
```

Comment:
Oh, no, I'm saying that `forceSelection` can keep its comma (as `inlineStyleOverride` did previously).